nebula/ui/components/VPNLinkButton.qml

```diff
@@ -11,7 +11,7 @@ import Mozilla.VPN 1.0
 VPNButtonBase {
     id: root
 
-    property var labelText
+    property var labelText: ""
```

Comment:
Can we use `string` as type instead?

Reply:
this is actually a good point, I believe using the correct data type not only provides a slight performance boost, but also allows you to not need to specify an `""` to denote that the string is defined, but empty

Reply:
Do you think it makes sense to audit the use of `var` and replace them with proper types? (not in this PR, not in this jira issue)

Reply:
It doesn't seem to be documented anywhere just how much of a performance degradation this causes, but it's enough for [the syntax checker look for them (id: M311)](https://doc.qt.io/qtcreator/creator-checking-code-syntax.html). That and we won't run into the QML warning that this PR sets out to fix as much. I tend to only use `var` when none of the other basic types would suffice

Reply:
> allows you to not need to specify an `""` to denote that the string is defined, but empty

Nice. updated. +1 to audit.